net.mancubus.SLADE.yaml

```diff
@@ -14,7 +18,6 @@ finish-args:
 
 # GZDoom requirements
 - --env=DOOMWADDIR=/app/share/games/doom
-- --persist=.config/gzdoom
```

Comment:
Out of curiosity, why was this switch removed? Is it no longer necessary?

Reply:
Well, it's only confusing since we actually have `--filesystem=home` for the whole application.

This causes an issue where users are told to use `~/.config/gzdoom` for their files, but the application actually looks in `~/.var/app/net.mancubus.SLADE/.config/gzdoom`. I discovered this discrepancy after reinstalling my computer, so I figured it would be better to fix it.

Do you agree? I can of cause change it back if you want.

Reply:
I see, thanks. Feel free to leave this as-is :slightly_smiling_face: